compiler/rustc_expand/src/base.rs

```diff
@@ -1229,7 +1229,7 @@ pub fn expr_to_spanned_string<'a>(
             ast::LitKind::ByteStr(_) => {
                 let mut err = cx.struct_span_err(l.span, err_msg);
                 err.span_suggestion(
-                    expr.span.shrink_to_lo(),
+                    expr.span,
```

Comment:
I cheated a bit here and simply changed the span to the whole expression.

Reply:
Ah, the shrink makes sense, but we need to make it one char wide instead of zero.. the whole span is wrong, as that would remove the entire string

Reply:
Ah, thank you

I wasn't sure how to make the span one char wide. That's why I shrinked it to `lo` and added `BytePos(1)`.

````
let span = expr.span.shrink_to_lo();
                err.span_suggestion(expr.span.shrink_to_lo(), span.with_hi(span.lo() + BytePos(1)), ...
```